src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int FIGURE_PARAMETER = 10;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final String color = colorSupplier.getRandomColor();
+        int index = random.nextInt(FIGURE_COUNT);
+        int rndFigureParameter = random.nextInt(FIGURE_PARAMETER);
```

Comment:
You are using the same side for each figure, it's not random
Also, rndFigureParameter - bad naming